gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/java/io/gravitee/repository/jdbc/management/JdbcApiKeyRepository.java

```diff
@@ -377,6 +377,35 @@ public List<ApiKey> findByApplication(String applicationId) throws TechnicalExce
         }
     }
 
+    @Override
+    public List<ApiKey> findByKeyAndEnvironmentId(String key, String environmentId) throws TechnicalException {
+        LOGGER.debug("JdbcApiKeyRepository.findByKeyAndEnvironmentId({}, {})", key, environmentId);
```

Comment:
maybe we should not log the key. That's why I have proposed ***** in my previous comment